app/src/main/java/com/marcossalto/targetmvd/network/models/Target.kt

```diff
@@ -0,0 +1,29 @@
+package com.marcossalto.targetmvd.network.models
+
+import com.marcossalto.targetmvd.models.TargetModel
+import com.marcossalto.targetmvd.models.TopicModel
+import com.squareup.moshi.Json
+
+data class Target(
+    @Json(name = "id") val id: Long = 0,
+    @Json(name = "title") val title: String = "",
+    @Json(name = "lat") val lat: Double = 0.0,
+    @Json(name = "lng") val lng: Double = 0.0,
+    @Json(name = "radius") val radius: Double = 0.0,
+    @Json(name = "topic_id") val topic_id: Int = 0
```

Comment:
perhaps the variable name should use camel case, being topicId instead of topic_id

Reply:
I just noticed you made this change in many places, is there a reason for the change?

Reply:
Thanks for your suggestion

Reply:
> I just noticed you made this change in many places, is there a reason for the change?

This change was related to KotlinJsonAdapter in Moshi. I fixed it